data/scenarios/Challenges/Ranching/fishing.yaml

```diff
@@ -0,0 +1,510 @@
+version: 1
+name: Fishing trip
+author: Karl Ostmo
+description: |
+  Catch a the appropriate seafood to make lunch.
+
+  If you haul any trash out of the lake, you
+  must throw it away in the enclosure to the southeast.
+  No littering!
+creative: false
+seed: 0
+attrs:
+  - name: shark
+    bg: "#2040bb"
+objectives:
```

Comment:
The objectives are quite confusing.  After completing the scenario by disposing of everything properly, I was confused why I still had not achieved the optional "no littering" objective.  "But surely I did not litter!" I thought.  "Why does it still think I have not completed that objective?"  I thought perhaps everything had to be hauled away, so I spent some time doing more fishing until I had the perfect number of junk items so that the hauler could haul them away, but I still did not complete the objective.

It is only when I looked at this `.yaml` file that I realized that objective is "completed" when you *do* litter, and that completing it is actually *bad*!  Do we have a way yet to mark an objective as "bad"/negative and display it with a different color / in a different section?  I thought we had an issue for that but I can't find it now.  In any case maybe just some text in the objective description would help that says something like "Note, this is an "anti-objective": it will be marked "completed" if you litter, which will cause you to fail the scenario.  Don't litter!"

Even better, maybe the description for the No Littering goal can be moved into the Sushi chef goal description, and the No littering goal can be marked hidden.  I think that would be a lot less confusing.

Reply:
> Do we have a way yet to mark an objective as "bad"/negative and display it with a different color / in a different section? I thought we had an issue for that but I can't find it now.

Closest issue may be #1046.

Agreed that the goal presentation should be improved.  Will look into it.